cmd/notation-hc-vault/key-helper/importKey.go

```diff
@@ -0,0 +1,176 @@
+package key_helper
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"github.com/google/tink/go/kwp/subtle"
+	vault "github.com/hashicorp/vault/api"
+	notationx509 "github.com/notaryproject/notation-core-go/x509"
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+var vaultAddr string
+
+func init() {
+	rootCmd.AddCommand(importKeyCmd)
+	importKeyCmd.PersistentFlags().String("key_path", "", "absolute path to the private key file")
+	importKeyCmd.PersistentFlags().String("cert_path", "", "absolute path to the certificate chain file")
+	importKeyCmd.PersistentFlags().String("key_name", "", "name of the key")
+
+}
+
+var importKeyCmd = &cobra.Command{
+	Use:   "import",
+	Short: "import - a simple CLI to import key and certificates to HashiCorp Vault",
+	Long: `import - a simple CLI to import key and certificates to HashiCorp Vault
+   
+import key to Vault Transit secrets engine and certificates to Vault KV secrets engine`,
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := context.Background()
+		fmt.Println("Import key start")
+		keyPath, err := cmd.Flags().GetString("key_path")
+		if err != nil {
+			fmt.Println(err)
+		}
+		certPath, err := cmd.Flags().GetString("cert_path")
+		if err != nil {
+			fmt.Println(err)
+		}
+		keyName, err := cmd.Flags().GetString("key_name")
+		if err != nil {
+			fmt.Println(err)
+		}
+		vaultClient, err := getVaultClient(ctx)
+		if err != nil {
+			fmt.Println(err)
+		}
+		fmt.Println("Successfully got vault client")
+		wrappingKey, err := getWrappingKey(ctx, vaultClient)
+		if err != nil {
+			fmt.Println(err)
+		}
+		fmt.Println("Successfully got wrapping key")
+		ciphertext, err := wrapPrivateKey(wrappingKey, keyPath)
+		if err := importKeyToTransit(ctx, vaultClient, ciphertext, keyName); err != nil {
+			fmt.Println(err)
+		}
+		fmt.Println("Successfully imported key to transit")
+		if err := importCertToKV(ctx, vaultClient, certPath, keyName); err != nil {
+			fmt.Println(err)
+		}
+		fmt.Println("Successfully imported cert to kv")
+	},
+}
+
+func getVaultClient(ctx context.Context) (*vault.Client, error) {
+	// read addr and token from environment variables
+	vaultAddr = os.Getenv("VAULT_ADDR")
+	if len(vaultAddr) < 1 {
+		log.Fatal("Error loading vault address")
```

Comment:
Instead of `log`, let's return an error here. Same applies to other `log` in this PR.
```suggestion
		return nil, errors.New("error loading vault address")
```